.github/CONTRIBUTING.md

```diff
@@ -123,12 +123,126 @@ Before creating a pull request, please read our requirements that explains the m
 
 #### Preview Changes
 
+##### Previewing in this repository
+
 1. Run `npm start` from within the `core` directory.
 2. A browser should open at `http://localhost:3333/`.
 3. From here, navigate to one of the component's tests to preview your changes.
 4. If a test showing your change doesn't exist, [add a new test or update an existing one](#modifying-tests).
 5. To test in RTL mode, once you are in the desired component's test, add `?rtl=true` at the end of the url; for example: `http://localhost:3333/src/components/alert/test/basic?rtl=true`.
 
+##### Previewing in an external app
+
+We can use `npm pack` to test Ionic Framework changes in an external app outside of this repository. Follow the below steps based on your framework.
+
+###### JavaScript
+
+Run the following commands to build the core directory and pack the changes:
+
+```bash
+cd core
+npm i
+npm run build
+npm pack --pack-destination ~
```

Comment:
Will these instructions work on Windows too? Alternatively, we could remove `--pack-destination ~` and use `[PATH TO TGZ]` or something in the `npm install` step below if the steps don't work on Windows.

Reply:
Good point! I will need to fire up the Windows machine to test this

Reply:
Sorry for making you fire up the Windows machine 😂

Reply:
If it helps, there's a bash terminal that I think comes packaged with Git, and VS Code has it built-in. I'm on a Windows machine myself and I do occasionally have to run scripts in bash for them to work, so there's precedent for that.